packages/babel-plugin-proposal-decorators/test/fixtures/2020-09-setters/options.json

```diff
@@ -0,0 +1,11 @@
+{
+  "plugins": [
+    [
+      "proposal-decorators",
+      { "version": "2020-09", "decoratorsBeforeExport": false }
+    ],
+    "proposal-class-properties",
+    "proposal-private-methods",
+    "proposal-class-static-block"
+  ]
+}
```

Comment:
Can you copy all these tests to also see how the transform behaves when only compiling decorators?